### ISSUE TITLE
Trailing slash in Cloud server url causes " Unable to generate configuration" error

### ISSUE THREAD

**Issue by darkvertex** [ankitpokhrel/jira-cli#264]
**Describe the bug**

Doing `jira init` then going with "Cloud" and a URL with a trailing slash (ie `https://company.atlassian.net/`) causes an error:

```
$ jira init
? Installation type: Cloud
? Link to Jira server: https://fpstudio.atlassian.net/
? Login email: alan@felixandpaul.com
? Default project: PP
? Default board: Pipeline Board
⠋ Creating new configuration...
✗ Unable to generate configuration: Config File "config" Not Found in "[]"
```
then if I do the same but no trailing slash, it works:
```
$ jira init
? Config already exist. Do you want to overwrite? Yes
? Installation type: Cloud
? Link to Jira server: https://<redacted>.atlassian.net
? Login email: alan@<redacted>.com
? Default project: PP
? Default board: Pipeline Board
⠼ Configuring metadata. Please wait...
✓ Configuration generated: /home/alan/.config/.jira/.config.yml
```

**Please provide following details**
1. JiraCLI Version:
    ```
    (Version="v0.3.0", GitCommit="", CommitDate="", GoVersion="go1.17.5", Compiler="gc", Platform="linux/amd64")
    ```
2. Are you using Jira cloud or on-premise jira server? Also mention the version for on-premise installation.
    ```
    Jira Cloud -- no idea where to see the version.
    ```
3. What operating system are you using? Also mention version.
    ```
    Ubuntu 20.04 LTS
    ```
4. What terminal are you using? Also mention version.
    ```
    Windows Terminal Preview
    Version: 1.12.3472.0
    ```

**To Reproduce**

1. export your JIRA_API_TOKEN env var then `jira init`
2. Pick `Cloud`
3. Enter server URL with a slash at the end and continue with your email.
4. Pick project and board.
5. See error.

**Expected behavior**
I expect URLs with slashes not to fail.

**Screenshots**

![image](https://user-images.githubusercontent.com/941331/149391938-047d95e1-bea7-4e51-a59c-68c927c2dff8.png)

**Additional context**

Seems similar bug to #255 (which was closed so I made a new ticket.)

**Comment by angristan** [comment]
I don't think the trailing slash is the cause of the issue. 

```
➜  ~ jira init
? Installation type: Local
? Link to Jira server: https://jira.domain.tld
? Login username: xxx
? Default project: xxx
? Default board: xxx
⠋ Creating new configuration...
✗ Unable to generate configuration: Config File "config" Not Found in "[]"
```

Second time with the exact same details:

```
➜  ~ jira init
? Config already exist. Do you want to overwrite? Yes
? Installation type: Local
? Link to Jira server: https://jira.domain.tld
? Login username: xxx
? Default project: xxx
? Default board: xxx
⠋ Creating new configuration...
✓ Configuration generated: /Users/stanislas/.config/.jira/.config.yml
```

**Comment by pmmalinov01** [comment]
```❯ jira init                                                                                                           13:45:06
? Installation type: Local
? Link to Jira server: https://FOO
? Login username: xxxx
? Default project: SRE
? Default board: SRE Kanban Planning
⠋ Creating new configuration...
✗ Unable to generate configuration: Config File "config" Not Found in "[]"```
I hit the same, couple of times.

**Comment by ankitpokhrel** [comment]
This is possibly fixed with #275

It would be great if someone can confirm this with the latest build (`brew install jira-cli --HEAD` or `go install github.com/ankitpokhrel/jira-cli/cmd/jira@main`)

**Comment by jaeyoung-cho** [comment]
Hi @ankitpokhrel 
I reproduced same situation like this issue on my computer.
```
jira init
? Installation type: Local
? Link to Jira server: https://rgpkorea.atlassian.net
? Login username: bottingtonnim@gmail.com
? Default project: PROL
? Default board: Project-L Board
⠋ Creating new configuration...
✗ Unable to generate configuration: Config File "config" Not Found in "[]"
```

So, I uninstalled jira-cli and reinstall it by your comment.
And then this issue was solved.

```
jira init
? Config already exist. Do you want to overwrite? Yes
? Installation type: Cloud
X Sorry, your reply was invalid: not a valid URL
? Link to Jira server: https://rgpkorea.atlassian.net
? Login email: bottingtonnim@gmail.com
? Default project: PROL
? Default board: Project-L Board
⠋ Creating new configuration...
✓ Configuration generated: /Users/a201808045/.config/.jira/.config.yml
```

**Comment by James-Firth** [comment]
I just tried initiatlizing this for the first time. I got an empty file initially and the error.

When I redid it after that and said I wanted to overwrite _then_ it created the config properly

```
➜ jira version
(Version="0.3.0", GitCommit="ea53355d5b72994a5b17a63e6e800ce9d8a23f4e", CommitDate="2022-01-09T19:50:19+00:00", GoVersion="go1.17.5", Compiler="gc", Platform="darwin/amd64")
```

**Comment by ankitpokhrel** [comment]
@James-Firth @mavaddat This issue is fixed in `main` branch however not released yet. I will make a new release as soon as I can but if you want to try unreleased version you can use `brew install jira-cli --HEAD` with homebrew or `go install github.com/ankitpokhrel/jira-cli/cmd/jira@main` with go.

**Comment by James-Firth** [comment]
@ankitpokhrel Awesome thanks for the detailed explanation and the hard work!

**Comment by aberezin** [comment]
I confirmed the fix worked by installing @main

**Comment by artemhp** [comment]
I also was able to reproduce this issue after fresh install according to manual in README.
This solution resolved for me this problem:
```
brew unlink jira-cli
brew install jira-cli --HEAD
jira init
```

**Comment by ankitpokhrel** [comment]
This fix is available in the new release [v1.0.0-beta](https://github.com/ankitpokhrel/jira-cli/releases/tag/v1.0.0-beta)
